examples/cpp_handler/README.md

```diff
@@ -0,0 +1,82 @@
+#### C++ based handler for TorchScript Model using DenseNet161 image classifier
+
+## How do I invoke C++ API from Python based program?
+Since handler runs in Python worker process, in order to invoke C++ API, you need to provide the Python torch_cpp_python_bindings
```

Comment:
@maheshambule Can you please explain why we need the Python worker to invoke the C++ API? The whole purpose of introducing support for C++ backend is for performance for fast inference. Why is it not possible to call C++ api directly from the Java FE Rest APIs?